### PR TITLE
Update 3.14.0 release notes to highlight login improvements

### PR DESCRIPTION
**Pull request by mrjones-plip**
per [slack thread](https://medic.slack.com/archives/C0813CQRX/p1644593917919739?thread_ts=1644536451.786769&cid=C0813CQRX):

> This appears in the release notes under "bug fixes" - but be aware that 3.14 contains the fix for [7343](https://github.com/medic/cht-core/issues/7343) which is the "mysterious logout bug" which seems quite impactful across projects
> 
> I might suggest we highlight that one in the release notes. Together with [6338](https://github.com/medic/cht-core/issues/6338) and [7187](https://github.com/medic/cht-core/issues/7187) there appears to be a theme in the release around "improved login experience"

### CODE REVIEW

**Comment by mrjones-plip** [comment]
Adding @latin-panda too - who ever gets it first ;)

File: content/en/core/releases/3.14.0.md [hunk 1]
```diff
@@ -80,6 +80,14 @@ We have made several improvements to our [server-side purging]({{< ref "apps/gui
 
 [#7280](https://github.com/medic/cht-core/issues/7280): Server Side Purge does not complete to the end
 
+
+### Improved login experience
+
+Users who have experienced issues with login should note that this release fixes some bugs. Specifically:
+- [#7343](https://github.com/medic/cht-core/issues/7343) Deployments using a browser on mobile shouldn't get erroneously logged out .
```
Comment (latin-panda) [inline content/en/core/releases/3.14.0.md:87]:
```suggestion
- [#7343](https://github.com/medic/cht-core/issues/7343) Deployments using a browser on mobile shouldn't get erroneously logged out.
```

File: content/en/core/releases/3.14.0.md [hunk 2]
```diff
@@ -80,6 +80,14 @@ We have made several improvements to our [server-side purging]({{< ref "apps/gui
...
+- [#6338](https://github.com/medic/cht-core/issues/6338) Changes to pseudo-static assets, like images and translations, won't cause unnecessary reloads for app users. 
+- [#7187](https://github.com/medic/cht-core/issues/7187)  Infinite loops and empty modal boxes at login should not longer be present.
```
Comment (latin-panda) [inline content/en/core/releases/3.14.0.md:89]:
```suggestion
- [#7187](https://github.com/medic/cht-core/issues/7187) Infinite loops and empty modal boxes at login should not longer be present. 
```

**Review by latin-panda — APPROVED** [review]
Thank you!

**Review by latin-panda — APPROVED** [review]
Cool!

**Pull request merged**